src/pip/_vendor/vendor.txt

```diff
@@ -22,4 +22,4 @@ setuptools==44.0.0
 six==1.14.0
 webencodings==0.5.1
 
-git+https://github.com/sarugaku/resolvelib.git@fbc8bb28d6cff98b2#egg=resolvelib
```

Comment:
Can we get a resolvelib release so that we include a released version of resolvelib in the pip 20.1?

Reply:
Just drop a word when you think it’s time, and I’ll cut a release. 😄